palisade-service/src/main/java/uk/gov/gchq/palisade/service/palisade/model/AuditablePalisadeResponse.java

```diff
@@ -29,18 +29,18 @@
  * Under normal conditions only one of these will be non-null, indicating failed or successful processing.
  */
 
-public final class AuditablePalisadeRequest {
+public final class AuditablePalisadeResponse {
 
     @JsonProperty("palisadeRequest")
```

Comment:
```suggestion
    @JsonProperty("palisadeSystemResponse")
```
This should be updated to remove any cause for confusion